Web_Development/LightFlow/lightflow/cli/main.py

```
@@ -0,0 +1,326 @@
+    try:
+        checkpoint_manager = CheckpointManager()
+        
+        if checkpoint:
+            # Show specific checkpoint info
+            checkpoint_info = checkpoint_manager.get_checkpoint_info(checkpoint)
+            click.echo("Checkpoint Information:")
+            for key, value in checkpoint_info.items():
+                click.echo(f"  {key}: {value}")
+        else:
+            # Show latest checkpoint
+            checkpoint_data = checkpoint_manager.load_checkpoint(workflow_name)
```

Ruff (0.12.2)

Local variable `checkpoint_data` is assigned to but never used

Remove assignment to unused variable `checkpoint_data`

(F841)